modules/amxBidAdapter.js

```diff
@@ -478,11 +511,51 @@ export const spec = {
       return;
     }
 
-    trackEvent('pbto', {
-      A: timeoutData.bidder,
-      bid: timeoutData.bidId,
-      a: timeoutData.adUnitCode,
-      cn: timeoutData.timeout,
+    let common = null;
+    const events = timeoutData.map((timeout) => {
+      const params = timeout.params || {};
+      const size = getTimeoutSize(timeout);
+      const { domain, page, ref } =
+        timeout.ortb2 != null && timeout.ortb2.site != null
+          ? timeout.ortb2.site
+          : {};
+
+      if (common == null) {
+        common = {
+          do: domain,
+          u: page,
+          U: getUIDSafe(),
+          re: ref,
+          V: '$prebid.version$',
+          vg: '$$PREBID_GLOBAL$$',
+        };
+      }
+
+      return {
+        A: timeout.bidder,
+        mid: params.tagId,
+        a: params.adunitId || timeout.adUnitCode,
+        bid: timeout.bidId,
+        aud: timeout.transactionId,
+        w: size[0],
+        h: size[1],
+        cn: timeout.timeout,
+        cn2: timeout.bidderRequestsCount,
+        cn3: timeout.bidderWinsCount,
+      };
+    });
+
+    const payload = JSON.stringify({ c: common, e: events });
+    const url = POST_TRACKING_ENDPOINT + '/g_pbto';
+
+    if (navigator.sendBeacon != null) {
+      navigator.sendBeacon(url, payload);
```

Comment:
You have to use core methods for network communication. If sendBeacom is useful, feel free to write a core method

Reply:
that's excellent, ty!!